.circleci/config.yml

```diff
@@ -98,24 +98,27 @@ jobs:
 
   unit_test_ios:
     executor: macos-executor
+    parameters:
+      plugin:
+        type: string
     steps:
       - install_flutter
       - checkout
       - install_melos
       - run:
           name: Run lint checks
-          command: melos run lint:ios --no-select
+          command: melos run lint:ios:<< parameters.plugin >>
       - run:
           name: Pre-start iOS simulator
           # xcrun instruments returns non zero, but successfully starts the simulator
           command: xcrun instruments -w "iPhone 11 (14.5) [" || true
       - run:
           name: Build example iOS apps
           command: |
-            melos run build:examples:ios
+            melos run build:examples:ios:<< parameters.plugin >>
       - run:
           name: Run iOS Unit Tests
-          command: melos run test:unit:ios
+          command: melos run test:unit:ios << parameters.plugin >>
       - run:
           name: Upload iOS test coverage
           command: melos run upload:coverage:ios
```

Comment:
Assuming this coverage upload still works as expected when it's only uploading a single job at a time?

Reply:
I think this was broken on Android anyway (you can see the commented out code in the bash script)